op-node/rollup/driver/state.go

```diff
@@ -367,6 +375,15 @@ func (s *Driver) eventLoop() {
 			s.derivation.Reset()
 			s.metrics.RecordPipelineReset()
 			close(respCh)
+		case respCh := <-s.startSequencer:
+			s.log.Info("Sequencer has been started")
+			s.driverConfig.SequencerEnabled = true
+			sequencingPlannedOnto = eth.BlockID{}
```

Comment:
without this, when restarting the sequencer I was running into this error:
https://github.com/ethereum-optimism/optimism/blob/d8e328ae936c6a5f3987c04cbde7bd94403a96a0/op-node/rollup/derive/batch_queue.go#L160

Reply:
See https://github.com/ethereum-optimism/optimism/pull/4367

This sequencer state should get cleaner and better separated with that, making it easier to handle here.

Reply:
Are you planning on prioritizing #4367 @protolambda? Can this PR get in before or after?

Reply:
This PR can get in first. There will be some tweaks to the driver loop to better separate the sequencing code, but the functionality will stay mostly the same. Can't guarantee that for future admin RPC changes though, there are more things left to do in that area.